SysKit.XCellKit/SpreadsheetCell.cs

```diff
@@ -110,9 +110,16 @@ public virtual void WriteCell(OpenXmlWriter writer, int columnIndex, int rowInde
                 {
                     sValue = sValue.Substring(0, 32767);
                 }
+
+                // From xml spec valid chars: 
+                // #x9 | #xA | #xD | [#x20-#xD7FF] | [#xE000-#xFFFD] | [#x10000-#x10FFFF]     
+                // any Unicode character, excluding the surrogate blocks, FFFE, and FFFF.
+                string re = @"[^\x09\x0A\x0D\x20-\uD7FF\uE000-\uFFFD\u10000-\u10FFFF]";
+                sValue = Regex.Replace(sValue, re, "");
```

Comment:
maybe just in case test out how fast is it if we were to export 1M rows before/and after the fix.

Reply:
There are differences, as expected, but not too much:
Run | Before[sec] | After[sec] | Diff[sec]
-- | -- | -- | --
1 | 51,21 | 55,0626 | 3,8526
2 | 52,6139 | 54,5115 | 1,8976
3 | 50,4388 | 52,1977 | 1,7589
**Avg** | 51,4209 | 53,923933 | **2,503033**

For 1M rows, I believe it is acceptable.